### DISCUSSION TITLE
与trojan相比速度下降，cpu占用高10倍！？

### DISCUSSION THREAD

**Discussion by iceyheart** [apernet/hysteria#308] [Q&A]
硬件背景：VPS甲骨文ARM 4C+24G，路由N1，上海移动200M宽带
1）trojan协议：fast.com测速下行280Mbps，上行45Mbps，VPS和N1CPU占用皆在5%上下；
2）Hysteria协议：fast.com测速下行62Mbps，上行120Mbps，VPS和N1 CPU占用皆在50%以上。
求解。

**Comment by tobyxdd** [comment]
配置里的速度写对了吗

**Reply by iceyheart** [reply]
{
  "listen": ":36712",
  "protocol": "udp",
  "obfs": "***", 
  "cert": "",
  "key": "",
  "up_mbps": 300,
  "down_mbps": 300,
  "resolver": "1.1.1.1:53",
  "resolve_preference": "64"
}

**Reply by tobyxdd** [reply]
"上海移动200M宽带" 为什么速度写 300

**Reply by tobyxdd** [reply]
而且你是对等 200M?

**Reply by tobyxdd** [reply]
ARM 用的是 arm64 吗

**Reply by iceyheart** [reply]
1、这是服务器的配置，服务器的带宽可达1G以上。客户端设定UP50，DOWN250
2、关于上传速率大过下载我也觉得很疑惑，正常上传只有50MBPs，并且客户端配置了50，唯独hysteria测试的时候速度倒挂
3、甲骨文的ARM机，听说这个服务器芯片性能很强，至少在这个案例里与它完全无关。

**Comment by iceyheart** [comment]
![image](https://user-images.githubusercontent.com/40313247/166972028-2925dc94-171f-4e70-8fbf-0210b32c8315.png)
